### ISSUE TITLE
The subtitles obtained are underlined and crossed out

### ISSUE THREAD

**Issue by jeluchu** [crunchy-labs/crunchy-cli#402]
**Describe the bug**
When downloading an episode, the subtitles that are downloaded along with the episode are crossed out and underlined.

**To Reproduce**
Execute the command that I have indicated in the issue, and open a video player once downloaded activating the subtitles (in this case I put the Spanish ones), I have tried with a macOS player (Elmedia Player) and also with VLC and I get the same result, crossed out and underlined subtitles.
```
crunchy-cli archive --language-tagging ietf --include-chapters -m audio -a ja-JP -s es-ES https://www.crunchyroll.com/es/watch/GPWUK9KD8/in-the-nick-of-time-a-big-time-maverick-from-the-west
```

**Expected behavior**
The result would be subtitles without strikethrough and underlining.

**Screenshots**
![Captura de pantalla 2024-05-06 a las 13 12 20](https://github.com/crunchy-labs/crunchy-cli/assets/32357592/448c6bd9-8308-4f64-994b-9f908b9e94c3)

**Client (please complete the following information):**
 - OS: macOS
 - Version: crunchy-cli v3.6.0

**Comment by bytedream** [comment]
This a bug in the subtitle library that was newly introduced (https://github.com/crunchy-labs/crunchy-cli/discussions/401#discussioncomment-9319617). I already created a patch and only waiting for it to get merged. Until the patch is available, use [v3.5.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.2)

**Comment by jeluchu** [comment]
Perfect! Thank you very much for the quick response and for your work! ❇️

**Comment by bytedream** [comment]
The fix is now available in [v3.6.1](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.6.1)
